packages/agoric-cli/src/lib/format.js

```diff
@@ -78,3 +83,43 @@ export const fmtRecordOfLines = record => {
   );
   return `{\n${lineEntries.join(',\n')}\n}`;
 };
+
+/**
+ * @param {{ brands: import('./format').AssetDescriptor[], offers: Map<number, OfferStatus>}} state
+ * @param {Awaited<ReturnType<typeof makeAgoricNames>>} agoricNames
+ */
+export const offerStatusTuples = (state, agoricNames) => {
```

Comment:
A 1 or 2 line description of what this function does would help me review it more quickly, I think.

Reply:
ah. this just got moved and renamed.